packages/UI/src/components/organisms/cards/PositionCard/PositionCard.stories.tsx

```diff
@@ -13,10 +13,11 @@ const TemplatePositionCard: ComponentStory<typeof PositionCard> = (args) => (
 const content = {
   title: 'Web Developer',
   level: 'junior',
-  timeCommittment: '5 hrs per week',
+  commitmentHoursPerWeek: '5 hrs per week',
   imgUrl: 'https://picsum.photos/200/300',
-  role: ' We are looking for a junior web developer who is willing and eagerto learn various web technologies mainly in the field of React while working within a team of ambitious developers! Our goal is to designa platform that brings people of from a diverse range of specialties together to collaboratively design the best possible software solutions. This project provides a space for people to establish and develop their ideas into working projects with input from others.',
-  tags: ['unity', 'react', 'social', 'agile', 'digital media'],
+  description:
+    ' We are looking for a junior web developer who is willing and eagerto learn various web technologies mainly in the field of React while working within a team of ambitious developers! Our goal is to designa platform that brings people of from a diverse range of specialties together to collaboratively design the best possible software solutions. This project provides a space for people to establish and develop their ideas into working projects with input from others.',
+  skills: ['unity', 'react', 'social', 'agile', 'digital media'],
```

Comment:
why changing those? changing those can result in broken components that use the old props, please revert this change